demo/admin/src/App.tsx

```diff
@@ -157,10 +157,10 @@ class App extends React.Component {
                                                                                                     documentTypes={pageTreeDocumentTypes}
                                                                                                     editPageNode={EditPageNode}
                                                                                                     category={category}
-                                                                                                    renderContentScopeIndicator={(scope) => (
+                                                                                                    renderContentScopeIndicator={(scope, values) => (
```

Comment:
Couldn't the scope indicator directly access `const { values } = useContentScope()` instead of passing it around?

Reply:
True, done